src/help/zaphelp/contents/credits.html

```diff
@@ -11,59 +11,60 @@
 
 <H2>ZAP Core Team (current)</H2>
 People who have made significant enhancements to the latest releases.<br>
-For more details see: https://www.ohloh.net/p/zaproxy/contributors
+For more details of code contributions see: <a href="https://www.ohloh.net/p/zaproxy/contributors">https://www.ohloh.net/p/zaproxy/contributors</a>
```

Comment:
Git is giving you trouble today...
https://www.openhub.net/p/zaproxy/contributors